torch/fx/passes/runtime_assert.py

```diff
@@ -350,27 +350,29 @@ def match_symbol(symint, cb):
                 and (sym_expr := _get_sym_val(node)) is not None
             ):
                 # this guards against deleting calls like item() that produce new untracked symbols
-                new_untracked_symbols = sym_expr.free_symbols - expr_to_proxy.keys()
+                def new_untracked_symbols():
+                    return sym_expr.free_symbols - expr_to_proxy.keys()
```

Comment:
Maybe do
```suggestion
                def has_new_untracked_symbols():
                    return bool(sym_expr.free_symbols - expr_to_proxy.keys())
```
to make it clear what it's used for